index.html

```diff
@@ -18,25 +18,19 @@
   <!-- <script src="./src/ts/index.ts"></script> -->
   <script defer type="module" src="dist\out-tsc\index.js"></script>
 </head>
-<body id="body">
-  <!--This is the navbar of the page-->
-  <nav>
-    <label class="logo"><span class="curved-line">ele</span>wa</label>
-    <ul>
-      <li><a class="" href="#">Solution</a></li>
-      <li><a href="#">Projects</a></li>
-      <li><a href="#">Company</a></li>
-      <li><a href="#">News</a></li>
-      <li><a class="nav-button" href="#">Contact Us</a></li>
-    </ul>
-    
-    <label id="icon">
-      <i class="fas fa-bars"></i>
-    </label>
-    <label id="icontwo">
-      <i class="fas fa-times"></i>
-    </label>
-  </nav>

```

Comment:
if this is from the main branch, was not supposed to be deleted since it might have been someone else's work, who created the navbar.